### PR TITLE
Implemented helpers and renamed "generateForkCards"

### LINKED ISSUE #47: generateForkCards does more than the name implies
## Description
generateForkCards generates cards and also renders them, in addition to this, it also wipes the wrapper. This level of functionality is greater than what the name implies.

## Acceptance Critera
- [x] Excessive functionality migrated to helper functions

### PR DESCRIPTION
**Pull request by te4-david-jensen**
## What issue did you implement or fix?
Fixed #47

## Summary
- Renamed function "generateForkCards"
- Implemented helpers

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

## How can this implementation be improved?
- Add comments and update documentation

### CODE REVIEW

**Review by te4-andre-skvarc — CHANGES_REQUESTED** [review]
Good work! :shipit: But I am missing the fundamental comments here to really understand what the code in depth until then I can't approve this. 🌵

File: public/js/main.js [hunk 1]
```diff
@@ -93,33 +93,50 @@ async function getRepos (userName) { return await fetchJSON(`/api/github/${userN
 
 async function getForks (userRepo) { return await fetchJSON(`/api/github/${userRepo}/forks`) }
 
-async function generateForkCards (forkList) {
-  resetWrapper()
+function loadForkCard (forkCard) {
```
Comment (te4-adrian-almetun-smeds) [inline public/js/main.js:96]:
```
function loadForkCard (forkCard) {
```
Should we not rename this function to reflect the fact it actually renders the cards. As opposed to `renderForkCards(data)` that does not implicitly render the cards but generates them and then render them. Can it be named something along the lines of ´handleForkCards` or something in that direction.

**Review by te4-adrian-almetun-smeds — CHANGES_REQUESTED** [review]
Overall it looks good. Just one comment regarding funcion naming.

**Review by te4-adrian-almetun-smeds — APPROVED** [review]
Looks good to me. 👍

**Review by te4-andre-skvarc — APPROVED** [review]
Amazing! Now we can understand this, even if we come back after 6 months away from the planet earth! 👍 :shipit:

**Pull request merged**